dds_web/config.py

```diff
@@ -64,6 +64,9 @@ class Config(object):
         "RATELIMIT_STORAGE_URI", "memory://"
     )  # Use in devel only! Use Redis or memcached in prod
 
+    # Redis
+    REDIS_URL = "redis://dds_redis"  # In production the URL also has a password
```

Comment:
The Redis SVC is `dds-redis`
```suggestion
    REDIS_URL = "redis://dds-redis"  # In production the URL also has a password
```

@rv0lt can you make this change so that we can test on dev instance

Reply:
Ok, I realise it shouldn't be `dds-redis`, but since we cannot have the same URL in all environments, it needs to handled via environment variable

Reply:
But for dev I think it is better to recreate the secret of the config? It has to be the same URL as RATELIMIT_STORAGE_URI uses there. And last time I checked it contains password details for the redis connection.

This one I setted it up as default due to being the one docker-compose uses, otherwise we would need to make modifications for local testing every time as well